app/models/ext_management_system.rb

```diff
@@ -603,6 +603,10 @@ def tenant_identity
     User.super_admin.tap { |u| u.current_group = tenant.default_miq_group }
   end
 
+  def top_level_manager
```

Comment:
I'm not sure I understand the question fully, but I was having to traverse up the tree from storage managers to cloud managers and you suggested this change in the parent PR.